### PR TITLE
translated output-pagination, output-sorting, helper-overview

### PR DESCRIPTION
**Pull request by cuileon**

### CODE REVIEW

File: docs/guide-zh-CN/helper-overview.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+全是静态类 （就是说它们只包含静态属性和静态方法，而且不能实例化）。
+
+可以通过调用其中一个静态方法来使用助手类，如下：
+
+```php
+use yii\helpers\Html;
+
+echo Html::encode('Test > test');
+```
+
```
Comment (samdark) [inline docs/guide-zh-CN/helper-overview.md]: Not translated?

File: docs/guide-zh-CN/helper-overview.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+> 注意：为了支持 [customizing helper classes](#customizing-helper-classes)，Yii 将每一个助手类
```
Comment (samdark) [inline docs/guide-zh-CN/helper-overview.md]: Not translated?

File: docs/guide-zh-CN/helper-overview.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+```php
+<?php
+
+namespace yii\helpers;
+
+class ArrayHelper extends BaseArrayHelper
+{
+    public static function merge($a, $b)
+    {
+        // your custom implementation
```
Comment (samdark) [inline docs/guide-zh-CN/helper-overview.md]: Not translated?

File: docs/guide-zh-CN/helper-overview.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    }
+}
+```
+
+将你的类保存在一个名为 `ArrayHelper.php` 的文件中。该文件可以在任何目录，例如 `@app/components`。
+
+接下来，在你的应用程序 [entry script](structure-entry-scripts.md) 处，在引入的 `yii.php` 文件后面
```
Comment (samdark) [inline docs/guide-zh-CN/helper-overview.md]: Not translated?

File: docs/guide-zh-CN/helper-overview.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+添加以下代码行，用 [Yii class autoloader](concept-autoloading.md) 来加载自定义类
```
Comment (samdark) [inline docs/guide-zh-CN/helper-overview.md]: Not translated?

File: docs/guide-zh-CN/output-pagination.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+分页
+==========
+
+当一次要在一个页面上显示很多数据时，通过需要将其分为
+几部分，每个部分都包含一些数据列表并且一次只显示一部分。这些部分被称为
+分页，例如 pagination。
+  
+如果你使用 [data provider](output-data-providers.md) 和 [data widgets](output-data-widgets.md) 中之一，
```
Comment (samdark) [inline docs/guide-zh-CN/output-pagination.md]: Not translated?

File: docs/guide-zh-CN/output-pagination.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+         'pages' => $pages,
+    ]);
+}
+```
+
+其次在视图中我们输出的模板为当前页并通过 pagination 对象链接到该页：
+
+```php
+foreach ($models as $model) {
+    // display $model here
```
Comment (samdark) [inline docs/guide-zh-CN/output-pagination.md]: Not translated?

File: docs/guide-zh-CN/output-sorting.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+         'models' => $models,
+         'sort' => $sort,
+    ]);
+}
+```
+
+在视图中：
+
+```php
+// display links leading to sort actions
```
Comment (samdark) [inline docs/guide-zh-CN/output-sorting.md]: Not translated?

File: docs/guide-zh-CN/output-sorting.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+echo $sort->link('name') . ' | ' . $sort->link('age');
+
+foreach ($models as $model) {
+    // display $model here
```
Comment (samdark) [inline docs/guide-zh-CN/output-sorting.md]: Not translated?

File: docs/guide-zh-CN/output-sorting.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
+排序
+=======
+
+有时显示数据会根据一个或多个属性进行排序。如果你正在使用
+[data provider](output-data-providers.md) 和 [data widgets](output-data-widgets.md) 中之一，排序
```
Comment (samdark) [inline docs/guide-zh-CN/output-sorting.md]: Not translated?

**Comment by cui-liang** [comment]
Sorry, the original translation missing several places, has supplemented completed.

File: docs/guide-zh-CN/output-pagination.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
+分页
+==========
+
+当一次要在一个页面上显示很多数据时，通过需要将其分为
+几部分，每个部分都包含一些数据列表并且一次只显示一部分。这些部分在网页上被称为
+分页。
+  
+如果你使用 [数据提供者](output-data-providers.md) 和 [数据小部件](output-data-widgets.md) 中之一，
+pagination 已经为你自动分页。否则，你需要创建 [[\yii\data\Pagination]]
+对象，为其填充数据例如 [[\yii\data\Pagination::$totalCount|total item count]]，
```
Comment (samdark) [inline docs/guide-zh-CN/output-pagination.md]: Not translated?

File: docs/guide-zh-CN/output-pagination.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+[[\yii\data\Pagination::$pageSize|page size]] 和 [[\yii\data\Pagination::$page|current page]]，在
```
Comment (samdark) [inline docs/guide-zh-CN/output-pagination.md]: Not translated?

File: docs/guide-zh-CN/helper-overview.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+可以通过调用其中一个静态方法来使用助手类，如下：
+
+```php
+use yii\helpers\Html;
+
+echo Html::encode('Test > test');
+```
+
+> 注意：为了支持 [自定义助手类](#customizing-helper-classes)，Yii 将每一个助手类
+  分隔成两个类：一个基类 (例如 `BaseArrayHelper`) 和一个 concrete 类 (例如 `ArrayHelper`).
```
Comment (samdark) [inline docs/guide-zh-CN/helper-overview.md]: Not translated?

File: docs/guide-zh-CN/helper-overview.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+  当使用助手类时，应该仅使用 concrete 类版本而不使用基类。
```
Comment (samdark) [inline docs/guide-zh-CN/helper-overview.md]: Not translated?

**Comment by cui-liang** [comment]
Done.

**Comment by samdark** [comment]
@qiansen1386 could you take a look?

**Comment by qiansen1386** [comment]
According to https://github.com/yiisoft/yii2/blob/master/docs/internals/translation-status.md, those 3 files are still under development yet, So that we did not put much efforts on those files.

About his translation, I think it is fairly good to be merged, we can do a little bit of polishing later on.

@cui-liang  舰长啊，你欠的RESTful的坑啥时候填啊。

**Comment by cui-liang** [comment]
@qiansen1386 这几篇我忘记是谁给我了，很长时间了，我周末的时候想起来就在这里发了个pr，以后我翻译的还是提交到你那里，你整理好了，再向官网发pr吧，这样太乱了！你那边要抓紧组织翻译了。

**Comment by samdark** [comment]
Merged. https://github.com/yiisoft/yii2/commit/3cbad7e8aea95bb72825e4d402500f7740a9cba3 Thank you!

**Pull request closed without merging**